src/PkgDeps.jl

```diff
@@ -5,26 +5,13 @@ using TOML: parsefile
 using UUIDs
 
 export PkgEntry, RegistryInstance
-export find_downstream_dependencies, reachable_registries
+export users, reachable_registries
 
 include("pkg_entry.jl")
 include("registry_instance.jl")
 
+const GENERAL_REGISTRY = "General"
 
-"""
-    _get_latest_version(base_path::AbstractString)
-
```

Comment:
why was this documentation deleted?

Reply:
I'm not sure how I feel about documenting internal functions yet, I can add it back in though.

Reply:
oh. why? documentation helps developers too. I think there are much better ways to keep something out of the public API than not documenting it.

Reply:
My grievances are around the format, I don't think that full docstrs matching public functions are needed for internal ones. But maybe something shortened, I'll bring it up in [BlueStyle](https://github.com/invenia/BlueStyle), and add this back in, as well as that for the others.